src/ui/Counter.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faMinusCircle, faPlusCircle } from '@fortawesome/free-solid-svg-icons'
+
+const CounterLabelStyle = styled.div`
+  width: 72px;
+  height: 24px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  border: 1px solid #c4c4c4;
+  border-radius: 6px;
+`
+
+const CounterStyle = styled.input`
+  margin: 0;
+  width: 20px;
+  border: none;
+  color: black;
+  font-weight: 500;
+  font-size: 0.75rem;
+  text-align: center;
+  font-family: 'Merriweather Sans', -apple-system, BlinkMacSystemFont,
+    'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans',
+    'Droid Sans', 'Helvetica Neue', sans-serif;
+`
+
+const CounterIconStyle = styled(FontAwesomeIcon)`
+  padding: 4px;
+  color: #c4c4c4;
+
+  :hover {
+    cursor: pointer;
+  }
+`
+
+function Counter({ counter, setCounter }) {
```

Comment:
We already have some components that store state (like `FoldablePanel` and `EllipsisMenu`), should we do the same here? Or it is a necessity to track the counter outside of the component?

Reply:
Yes, the way we implemented it the counter is tracked outside the component, it can be a state or something else.